languages/scala/reference/exercise-concepts/hamming.md

```diff
@@ -0,0 +1,32 @@
+# Concepts of hamming
+
+[Example implementation](https://github.com/exercism/scala/blob/master/exercises/hamming/example.scala)
+
+## General
+
+- object or case class: used as class wrapper for exercies
+- functions: used as main entry point for the exercise
+- function arguments: input strands passed as arguments
+- return values: return a value from a function
+- conditionals using if: conditionally execute logic using an `if` statement
+- strings: `string` passed as the two input parameters
+- character operations: `!=`
+- integer: `Int` is used as return result within `Option`
+- Option: `Option` is used as return result
+- math operations: `==`
+- zip: combine two strings into a sequence of tuples
```

Comment:
Tuples are mentioned here, so I think they should also be listed as a separate Concept.

Reply:
good point.